editoast/schemas/src/rolling_stock.rs

```diff
@@ -174,3 +175,77 @@ impl Serialize for RollingStock {
         RollingStock::serialize(self, serializer)
     }
 }
+
+impl RollingStock {
+    /// Returns the supported signaling systems, including "ETCS_LEVEL2" if `etcs_brake_params` is set.
+    pub fn get_all_supported_signaling_systems(&self) -> RollingStockSupportedSignalingSystems {
+        let mut supported_signaling_systems = self.supported_signaling_systems.0.clone();
+        if self.etcs_brake_params.is_some() {
+            supported_signaling_systems.push("ETCS_LEVEL2".into());
+        }
+        RollingStockSupportedSignalingSystems(supported_signaling_systems)
+    }
+
+    pub fn get_row_supported_signaling_systems(&self) -> RollingStockSupportedSignalingSystems {
```

Comment:
typo "r`a`w" + more idiomatic:
```suggestion
    pub fn raw_supported_signaling_systems(&self) -> RollingStockSupportedSignalingSystems {
```
Ditto for other occurrences.